setup.cfg

```diff
@@ -97,3 +99,12 @@ dev =
     types-psutil
     types-requests
     types-tabulate
+    # These were provided by orquestra-python-dev -
+    # see https://zapatacomputing.atlassian.net/browse/ORQSDK-964
+    black~=22.3
+    flake8>=4.0.0
+    Flake8-pyproject>=0.9.0
+    isort~=5.9.0
+    mypy~=0.910
+    pytest~=6.2
+    pytest-cov>=2.12
```

Comment:
can we bump any of these too?

For example, black has the 23 series.

I'd stay away from bumping pytest in this PR because who knows what will happen.
Mypy had v1, but that might also be a can of worms.

Reply:
Yeah, it should be fine to bump these.

I think the original plan was to switch back to `orquestra-python-dev` soon, so keeping these requirements relatively unchanged made sense. However, during the planning yesterday we agreed to decouple from `orquestra-python-dev` altogether, so we're free to bump our linters' versions.

Reply:
Bumping black caught one additional formatting error that I've fixed in [880b1e2](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/308/commits/880b1e2d97f91a91a525f6010fc7ddf420c35a9f).
Bumping mypy seemed to be fine, I got identical results with both 0.910 and 1.5.1